Dockerfile

```diff
@@ -3,4 +3,10 @@ LABEL description="Docker image containing all requirements for nf-core/openmspe
 
 COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
+RUN mkdir /opt/thirdparty && \
```

Comment:
We should consider packaging comet to bioconda (if this is not already existing at all) .

Reply:
yeah we were just discussing this.. and thought this might be the easiest fix for now. I think currently there is no comet bioconda package..

Reply:
yes that would be nice to have, however all thirdparty tools required by openms can be easily cloned from github thirdparty repository.. and there might be more coming in the future

Reply:
Hm, the "normal way" we deal with this is making these then available via bioconda as well. If they are easily found somewhere, one can simply create bioconda recipes for each of the ones used and push this on-demand to bioconda. That would make the overall process cleaner than cloning something inside the Dockerfile etc. 

The main reasoning behind this was, to get rid of these "custom" Dockerfiles per pipeline and just have a common process of Dockerfile + environment.yaml to load everything that is required.

Reply:
ok makes sense. Are you able to set up a bioconda package for comet? Or should I, not sure how to do this though..
http://comet-ms.sourceforge.net/